dep_updates.md

```diff
@@ -0,0 +1,56 @@
+# Sinopia Weekly Dependency Updates
+
+This is meant to be a guide to DLSS Infrastructure Team on how to do the weekly dependency update work for all the Sinopia related bits we maintain.
+
+## Which Repos?
+
+The repos that get weekly dependency updates:
+
+(from https://github.com/sul-dlss/access-update-scripts/blob/master/infrastructure/projects)
+- LD4P/sinopia_api
+- LD4P/sinopia_editor
+- LD4P/sinopia_exporter
+- LD4P/sinopia_indexing_pipeline
+
+## For Deployed Development Environment
+
+After the automated dependency update PRs are merged (via script, usually - see [First Responder doc](https://github.com/sul-dlss/DeveloperPlaybook/blob/master/infrateam_first_responder.md#merge-em)), CircleCI will automatically build a docker image and push it to DockerHub.
+
+(See https://github.com/sul-dlss/DevOpsDocs/blob/master/projects/sinopia/operations-concerns.md#development)
+
+## For Stage and Production Environments
+
+You do NOT need to create a github tagged release issue such as https://github.com/LD4P/sinopia_editor/issues/2755 for dependency updates.
```

Comment:
This isn't correct. We use the tagged releases for stage and production to generate the docker images used in those environments.

Reply:
@jermnelson I read this line as referring to creating a new *issue* for the release, which we haven't been doing for dep-update only releases.

Reply:
Yes, exactly.  This is one of the many points of confusion.  I put "issue" into italics to make it clearer.